pkg/util/proto/document.go

```diff
@@ -149,11 +154,13 @@ func (d *Definitions) parsePrimitive(s *openapi_v2.Schema, path *Path) (Schema,
 	}
 	switch t {
 	case String:
+		fallthrough
```

Comment:
I don't know which version I dislike the most :-)

Reply:
Yeah, I've now removed `fallthrough` and instead added a comment (`// do nothing`). Give how go doesn't fallthrough by default, a comment will help most of us that are used to how switch statements behave in other languages.